installer/windows/installrelease.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+rem NB: only installs from *first* release directory
```

Comment:
I'm not sure what is use case for this script.

Reply:
When I need to test a non-gui change to the storagenode or updater service or the installer, I run `make storagenode_windows_amd64` and/or `make storagenode-updater_windows_amd64` on the host OS followed by `build.bat` and `installrelease.bat` on the windows guest to get automated build and unattended installation of the HEAD.

The reason for the for loop is just to get into the release directory, whose name we don't know. Perhaps there's a more concise way to do this.

Reply:
My main issue here (and with `build.bat`) is that this script expects locations and files generated by docker and I don't plan to install docker on my Windows vm :) I will need to make `release` directory, directory inside it and copy binaries there with exact names. I would like to have one basic script that is very flexible and on top of it, we can build something more advanced like installing automatically from `make storagenode*` command.

Reply:
I take your point and of course we could break these down into smaller, more composable pieces. In the case of testing the installer and the updater in-particular, I found that version information is often required which is painstaking to do manually, hence my reaching for our make targets for building binaries.

Reply:
In fact, version information is now required to build the installer.

Reply:
But this script doesn't have to be worried about version information. Just try to build msi. If version will be missing it will fail.

Reply:
In the case of this file `installrelease.bat` the filename should make it clear that it's intended to be used in conjunction with our conventional release directory structure. `install.bat` is the more flexible, composable thing that you're talking about.

Reply:
yes, sure. We talked about everything in one thread and i think I didn't say clearly what I have on my mind. To summarize, I'm ok with `install.bat`, `installrelease.bat` (except my last comment about `/x`). I would just like to have `build.bat` script less dependent on docker environment paths.

Reply:
Done.